release notes/v0.51.0.md

```diff
@@ -6,10 +6,22 @@ k6 `v0.51.0` is here 🎉! This release includes:
 
 ## Breaking changes
 
-- `#pr`, `<small_break_1>`
-- `#pr`, `<small_break_2>`
+### Browser APIs to Async
 
-### (_optional h3_) `<big_breaking_change>` `#pr`
```

Comment:
Do not drop the placeholders, please

Reply:
I've added them back in 9d9e50b148f4e13780204c0cbfe416c85ac781fa